internal/characters/tighnari/skill.go

```diff
@@ -0,0 +1,54 @@
+package tighnari
+
+import (
+	"github.com/genshinsim/gcsim/internal/frames"
+	"github.com/genshinsim/gcsim/pkg/core/action"
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/combat"
+)
+
+var skillFrames []int
+
+const skillHitmark = 20
+
+func init() {
+	skillFrames = frames.InitAbilSlice(15)
+}
+
+func (c *char) Skill(p map[string]int) action.ActionInfo {
+	ai := combat.AttackInfo{
+		ActorIndex: c.Index,
+		Abil:       "Vijnana-Phala Mine",
+		AttackTag:  combat.AttackTagElementalArt,
+		ICDTag:     combat.ICDTagNone,
+		ICDGroup:   combat.ICDGroupDefault,
+		StrikeType: combat.StrikeTypeDefault,
+		Element:    attributes.Dendro,
+		Durability: 25,
+		Mult:       skill[c.TalentLvlSkill()],
+	}
+
+	c.Core.QueueAttack(
+		ai,
+		combat.NewCircleHit(c.Core.Combat.Player(), 2, false, combat.TargettableEnemy),
+		skillHitmark,
```

Comment:
when you add frames later, this thing should be snapping on the release frame and not when it hits the target